README.md

```diff
@@ -63,6 +63,10 @@ For example, on macOS with Homebrew: `brew install cmake pkg-config` and on Ubun
 
 See the [commands documentation](./docs/commands.md) for additional documentation, or run `licensed -h` to see all of the current available commands.
 
+### Automation
+
+Do `bundle plugin install bundler-licensed` to automatically run `licensed cache` after running `bundle install` or `bundle update` commands.
```

Comment:
What do you think about this?  I'd rather link your repo with generic information than include specific details here.  If the plugin changes that reduces the need to update this README, is transparent to users where the plugin source is, and is more clear that the plugin is not "owned" by this repo

```suggestion
The [bundler-licensed plugin](https://github.com/sergey-alekseev/bundler-licensed) runs `licensed cache` automatically when using `bundler`.  See the linked repo for usage and details.
```

Reply:
Makes sense.